### DISCUSSION TITLE
One serial connection not working

### DISCUSSION THREAD

**Discussion by S-A-M-J** [paukstelis/octoprint_deploy#69] [Q&A]
I've just used the script two create two instances. However one of them is unable to connect to the printer via serialport (receives no Answer). The setup worked flawlessly but after logging into the instance it is unable to connect. 

Printers: 2x Prusa mk3s+
Pi: Pi 4b
Os: Octopi

What i've tried: 
- Reinstalling the instance
- Reconnecting the printer
- Rebooting both pi and octoprint

**Comment by DanCantrell** [comment]
Have you tried a different USB cable? I am running three instances and a few weeks ago I had a print time out and fail because of the cheap USB cable I was using. It worked sometimes and sometime not. What I thought of when you said it worked for setup but not after. Word of advise: use good cables. The one that failed on me was a cheap Walmart brand with plastic housings. Get a good brand if you can like Cable Creation, Cable Matters or ANIOPE...Anything with metal ends will be better than the plastic ones..
